appinfo/info.xml

```diff
@@ -73,6 +73,7 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 			<step>OCA\Talk\Migration\CreateHelpCommand</step>
 			<step>OCA\Talk\Migration\ClearResourceAccessCache</step>
 			<step>OCA\Talk\Migration\CacheUserDisplayNames</step>
+			<step>OCA\Talk\Migration\ResetCustomPermissionsForModerators</step>
```

Comment:
Need to be repair step?
Isn't this a pontual change?

Reply:
yeah, shouldn't happen anymore after the code change, unless we miss something.

Reply:
Was the other steps created with the same intention to run only a time?

Reply:
No, they are "safety nets" to repair undesired admin actions, etc.